Alpine local encryption test is breaking

```
mkdir -p ~/.earthly/
  cat << EOF > ~/.earthly/config.yml
  global:
    buildkit_additional_config: |
      [registry."docker.io"]
        mirrors = ["registry.docker-mirror.svc.cluster.local:[5](https://github.com/kairos-io/kairos/actions/runs/9882563465/job/27295942578?pr=2714#step:9:5)000"]
      [registry."registry.docker-mirror.svc.cluster.local:5000"]
        insecure = true
        http = true
  EOF
  earthly -P +run-qemu-datasource-tests --PREBUILT_ISO=$(ls *ubuntu*.iso) --TEST_SUITE=reset-test --FLAVOR=ubuntu
```

results in 

```
**  +run-qemu-datasource-tests *failed* | • [FAILED] [1226.087 seconds]
  +run-qemu-datasource-tests *failed* | kairos reset test [BeforeEach] auto installs, reboots and passes functional tests resets [reset-test]
  +run-qemu-datasource-tests *failed* |   [BeforeEach] /test/tests/reset_test.go:14
  +run-qemu-datasource-tests *failed* |   [It] /test/tests/reset_test.go:33
  
  +run-qemu-datasource-tests *failed* |   [FAILED] Timed out after 1225.051s.
  +run-qemu-datasource-tests *failed* |   Expected
  +run-qemu-datasource-tests *failed* |       <string>: 
  +run-qemu-datasource-tests *failed* |   to equal
  +run-qemu-datasource-tests *failed* |       <string>: ping\n
  +run-qemu-datasource-tests *failed* |   In [BeforeEach] at: /go/pkg/mod/github.com/spectrocloud/peg@v0.0.0-20240405075800-c5da7125e30f/matcher/helpers.go:264 @ 07/11/24 00:19:25.982
  +run-qemu-datasource-tests *failed* | ------------------------------
  +run-qemu-datasource-tests *failed* | SSSSSSS
  
  +run-qemu-datasource-tests *failed* | Summarizing 1 Failure:
  +run-qemu-datasource-tests *failed* |   [FAIL] kairos reset test [BeforeEach] auto installs, reboots and passes functional tests resets [reset-test]
  +run-qemu-datasource-tests *failed* |   /go/pkg/mod/github.com/spectrocloud/peg@v0.0.0-20240405075800-c5da7125e30f/matcher/helpers.go:264
  
  +run-qemu-datasource-tests *failed* | Ran 1 of 21 Specs in 1226.088 seconds
  +run-qemu-datasource-tests *failed* | FAIL! -- 0 Passed | 1 Failed | 0 Pending | 20 Skipped
  +run-qemu-datasource-tests *failed* | --- FAIL: TestSuite (1226.09s)**
```

e.g. https://github.com/kairos-io/kairos/actions/runs/9882563465/job/27295942578?pr=2714

Also we don't run this test on master, so you need to open a PR to see it fail atm

--- a/.github/workflows/reusable-encryption-test.yaml
+++ b/.github/workflows/reusable-encryption-test.yaml
@@ -16,7 +16,7 @@ on:
 permissions: read-all
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: kvm
     defaults:
       run:
         shell: bash -eo pipefail {0}
@@ -97,7 +97,11 @@ jobs:
         env:
           LABEL: ${{ inputs.label }}
           EMULATE_TPM: true
+          CREATE_VM: true
           USE_QEMU: true
+          MEMORY: 4000
+          CPUS: 2
+          KVM: true
         run: |
           export ISO=$(ls $PWD/*${{ inputs.flavor }}*.iso)
           .github/encryption-tests.sh

--- a/.github/workflows/reusable-encryption-test.yaml
+++ b/.github/workflows/reusable-encryption-test.yaml
@@ -16,7 +16,7 @@ on:
 permissions: read-all
 jobs:
   test:
-    runs-on: kvm
+    runs-on: ubuntu-latest
     defaults:
       run:
         shell: bash -eo pipefail {0}
@@ -101,7 +101,6 @@ jobs:
           USE_QEMU: true
           MEMORY: 4000
           CPUS: 2
-          KVM: true
         run: |
           export ISO=$(ls $PWD/*${{ inputs.flavor }}*.iso)
           .github/encryption-tests.sh

